testament/testament.nim

```diff
@@ -70,6 +71,9 @@ type
     startTime: float
     spec: TSpec
 
+    inCurrentBatch: bool ## whether the test is part of the current
```

Comment:
very small thing, entirely ignorable: I _believe_, if the blank line above this one is removed github has an easier time correctly rendering context text, otherwise the context snippet goes all the way upto `  type` on line 57/58.

Reply:
It looks like that didn't help, but it was worth a try.